test/test_torch.py

```diff
@@ -299,6 +299,16 @@ def test_tan(self):
     def test_tanh(self):
         self._testMathByName('tanh')
 
+        # test with high values
+        x = np.linspace(-2000000000000.0, 2000000000000.0, 100)
+        y = torch.FloatTensor(x)
+        self.assertEqual(np.tanh(y), torch.tanh(y))
+
+        # test with fractional values
+        x = np.linspace(-1.0, 1.0, 100)
+        y = torch.FloatTensor(x)
+        self.assertEqual(np.tanh(y), torch.tanh(y))
```

Comment:
You need to make a separate test for the numpy part and guard it with a check for TEST_NUMPY